src/html-conventions-report.xsl

```diff
@@ -61,7 +61,7 @@
                     <xsl:apply-templates/>               
                 </main>
                 <footer>
-                    <p> something maybe here </p>
+                    <p> footer here </p>
```

Comment:
use this text:
"This document is generated automatically by [the model2owl tool](https://github.com/costezki/model2owl).
and is licenced under a [CC BY 4.0 licence](https://creativecommons.org/licenses/by/4.0/deed.en)"

Reply:
exactly what i needed